pyrobosim/pyrobosim/core/world.py

```diff
@@ -624,19 +625,15 @@ def remove_all_objects(self, restart_numbering=True):
         if restart_numbering:
             self.object_instance_counts = {}
 
-    def update_bounds(self):
+    def update_bounds(self, entity: Union[Room, Hallway]):
```

Comment:
This is the only place we would be using Python type hints... which is good, but maybe let's not do it in this PR?

Up to you if you still want to validate using `isinstance` or just assume that this method will only get called internally with a valid entity :)

Reply:
> This is the only place we would be using Python type hints... which is good, but maybe let's not do it in this PR?

It did feel a bit weird 
> Up to you if you still want to validate using isinstance or just assume that this method will only get called internally 
with a valid entity :)

Will go with validating using `isinstance`